src/game-of-life/game.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from 'react';
+import GameOfLife from "./game-of-life";
+import Square from "../square";
+
+export default class Game extends Component {
+    constructor(props) {
+        super(props);
+        this.gameOfLife = new GameOfLife(this.props.length);
+        this.state = {
+            grid: this.gameOfLife.initialize()
+        };
+        this.changeGridState = this.changeGridState.bind(this);
```

Comment:
remove this - change function changeGridState to arrow fun

Reply:
sorry this file shouldnt exists